src/Data/MongoDBMock/contests.js

```diff
@@ -0,0 +1,147 @@
+const contests = [
+  {
+    contestId: 'a0347677-c3c9-4edc-9d46-fed4a958fdc2',
+    contestName: 'XII Zawody im. Pana Starosty',
+    kynologiqueDepartment: 'Gdynia',
+    startDate: new Date(2022, 3, 22, 10, 0),
+    endDate: new Date(2022, 3, 23),
+    applicationOpenDate: new Date(2022, 1, 10, 10, 0),
+    applicationClosedDate: new Date(2022, 2, 20, 23, 59),
+    place: 'Gdańsk',
+    detailAdress: 'ul.Grunwaldzka 222',
+    judges: ['Alberto Makkaroni'],
```

Comment:
Could judges be users also? in this case you could also add user Id

Reply:
I think, at that moment it is not so likely. In life judges are mostly "forest granddads" :)) or from abroad